### ISSUE TITLE
Escape commas in std::vector<std::string> JOmnifactory parameters

### ISSUE THREAD

**Issue by ruse-traveler** [JeffersonLab/JANA2#380]
I recently encountered an issue working on `EICrecon` that seems to be related to JANA2 parsing `std::vector<std::string>` parameters of a JOmnifactory...

To resolve [eic/EICrecon#1669](https://github.com/eic/EICrecon/issues/1669), I modified an algorithm (see [here](https://github.com/eic/EICrecon/pull/1668)) to accept a vector of expressions stored as strings with the intent of providing a way to map one set of readout indices onto another. For example:
```
phi-fmod(phi,5)
```

However, the expression that gets passed to the algorithm is:
```
phi-fmod(phi
```

This seems to be JANA2 splitting on the `,` in the expression. Would it be possible to escape commas in strings (maybe like `fmod(phi\,5)`) so that the string is passed to the algorithm as a whole?

**Comment by nathanwbrei** [comment]
I agree it would be great to support this feature. I'll add it to the roadmap for the next release.

**Comment by ruse-traveler** [comment]
🙌 🙌 🙌
